packages/block-library/src/navigation-link/index.php

```diff
@@ -382,7 +382,7 @@ function register_block_core_navigation_link() {
  * @param array $settings Default editor settings.
  * @return array Filtered editor settings.
  */
-function gutenberg_enable_animation_for_navigation_link_inspector( $settings ) {
+function block_core_navigation_link_enable_inspector_animation( $settings ) {
```

Comment:
We can do this quick fix for now(renaming), but we need to see what we're going to do with `__experimentalBlockInspectorAnimation` introduced [here](https://github.com/WordPress/gutenberg/pull/46342/).

This setting is part of the [auditing of experimental API](https://github.com/WordPress/gutenberg/issues/47196) for `6.2`. @artemiomorales is this something that we want to stabilize or exclude from `6.2`?

If the plan is to remain experimental, among the changes that will be needed, these functions(for the two other blocks) would need to be moved in different files and not remain here.